axum-core/CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# 0.5.2
+
+- **added:** Implement `Stream::size_hint` for `BodyDataStream` ([#3195])
+- **change:** Some documentation improvements
```

Comment:
I don't see the benefit of mentioning this so I wouldn't include it. Feel free to ignore this though, it doesn't really harm anyone either.